src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -159,6 +153,14 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
     return future;
   }
 
+  public CompletableFuture<Integer> getPurchaseOptionCode(Object s){
```

Comment:
See above.  The implementation of this will eventually be replaced with something that parses the purchase option code and returns the numeric user limit